srcpkgs/ocean-sound-theme/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'ocean-sound-theme'
+pkgname=ocean-sound-theme
+version=6.4.0
+revision=1
+build_style=cmake
+hostmakedepends="extra-cmake-modules qt6-base"
+makedepends="qt6-base-private-devel"
+short_desc="Ocean Sound Theme for Plasma"
+maintainer="Rose <rose@pinkro.se>"
+license="CC-BY-SA-4.0"
+homepage="https://invent.kde.org/plasma/ocean-sound-theme"
+distfiles="${KDE_SITE}/plasma/${version}/${pkgname}-${version}.tar.xz"
```

Comment:
remove the `pkgname` variable, it not supposed to be variable when either upstream or the package name changes.